FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -18,7 +18,56 @@ public final class RemoteFeedLoader: FeedLoader {
 		self.client = client
 	}
 
+	private static let successCode = 200
+
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case .success((let data, let response)):
```

Comment:
The let can be defined outside the tuple so the expression can be shorter:

```suggestion
			case let .success((data, response)):
```